js/search/search.js

```diff
@@ -254,18 +254,21 @@ class Search {
                 return;
             }
 
-            let sd = new Date().getTime() - SEARCH_PERIOD_SUBTRACTOR;
-            let sd_time = MINIMUM_DATE;
-            if (startDate && startDate !== "" && typeof startDate === 'object') {
-                sd_time = new Date(startDate).getTime();
-                if (sd_time >= sd) {
-                    sd_time = sd;
+            let searchPeriod = new Date().getTime() - SEARCH_PERIOD_SUBTRACTOR;
+            let sd_time = searchPeriod;
```

Comment:
Please change the naming convention here. Underscore is ugly and the variable itself can be named to startDatePeriod. It's ok to have verbose names wherever necessary.